requirements/source/requirements-dev.in

```diff
@@ -4,4 +4,5 @@
 pip-tools==7.4.1
 beautifulsoup4<5
 pytest-playwright~=0.5.0
+django-jinja==2.10.2
```

Comment:
This is already in the base requirements, which this file extends

Reply:
oh yes, now removed.